rc-node-bindings.cc

```diff
@@ -259,7 +259,7 @@ namespace rc {
             return;
         }
         float duty = (float)info[0]->ToNumber()->Value();
-        rc_set_motor_all(duty);
+        rc_motor_set(0,duty);
```

Comment:
is this really the same as 'all'?

Reply:
I guess it is: http://strawsondesign.com/docs/librobotcontrol/group___motor.html#gaeaf337ad16953c9f9412d96a600eccc1